# Builds pixman as position-independent code if the `b_staticpic` option is enabled.

diff --git a/pixman/meson.build b/pixman/meson.build
index fd41288..b8b2dbd 100644
--- a/pixman/meson.build
+++ b/pixman/meson.build
@@ -126,7 +126,7 @@ libpixman = library(
   dependencies : [dep_m, dep_threads],
   include_directories : cpu_features_inc,
   version : meson.project_version(),
-  install : true,
+  pic : get_option('b_staticpic'),
 )
 
 inc_pixman = include_directories('.')
